minerl/Malmo/Schemas/MissionHandlers.xsd

```diff
@@ -1255,6 +1255,17 @@
         </xs:complexType>
     </xs:element>
 
+    <xs:element name="ServerQuitFromDeath">
```

Comment:
maybe <ServerTimelimt>? QuitFromDeath seems misleading, if this is supposed to be a timelimit

Reply:
Good catch